### PR TITLE
Added Win32 plattform

### PR DESCRIPTION
**Pull request by AlexHild**
PR adds a Vulkan.Win32 plattform project. Some minor changes in the generator were necessary to generate the code for Win32.

However, as vulkan dll is named vulkan-1 under windows wrapper does not work. Can this fixed by using dllmap under mono?

### CODE REVIEW

**Comment by radekdoulik** [comment]
Hi Alex,

thanks for your PR! I have been meanwhile working in the same area. Could you please update your changes and also fix white space formatting?

I am trying to use the dllmap for Vulkan project, it doesn't seem to work though so far.

We will also need a copyright assignment from you, which should have been asked to sign from you by CLA bot. Looks like the bot is down now. Let me look into it.

Radek

File: src/Platforms/Win32/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Vulkan.Win32")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Vulkan.Win32")]
+[assembly: AssemblyCopyright("Copyright ©  2016")]
```
Comment (radekdoulik) [inline src/Platforms/Win32/Properties/AssemblyInfo.cs]: We need this line to contain "Copyright (c) Xamarin Inc"

File: tools/Generator/Generator.cs [hunk 2]
```diff
@@ -16,7 +16,10 @@ public class Generator
 		bool isInterop;
 		bool needsMarshalling;
 
-		Dictionary<string, string> typesTranslation = new Dictionary<string, string> () { { "ANativeWindow", "IntPtr" }  };
+		Dictionary<string, string> typesTranslation = new Dictionary<string, string> () {
+            { "ANativeWindow", "IntPtr" },
+            { "HWND", "IntPtr" },
+            { "HINSTANCE", "IntPtr" },  };
```
Comment (radekdoulik) [inline tools/Generator/Generator.cs]: There's a white space issue (and in other places too)

**Comment by AlexHild** [comment]
Submitted #17 (hopefully) fixing all mentioned issues

**Pull request closed without merging**